### ISSUE TITLE
bash color issue after error

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tinselcity/skipfish#9]
```
What steps will reproduce the problem?
1. run skipfish without parameter
2. See the error : [-] PROGRAM ABORT : Scan target not specified (try -h
for help).
    Stop location : main(), skipfish.c:379
3. verify the color of your shell, it should be slightly different


Attached a patch that should correct the issue. I called the variable with
the "normal color" cRST for ReSeT. (being black, white or whatever the user
normally uses) . 
You might want to correct the #  define cNOR "\x1b[0;37m" variable, but I'm
not sure where you use that one.


```

Original issue reported on code.google.com by `christophevandeplas@gmail.com` on 20 Mar 2010 at 4:00

Attachments:
- [skipfish-color-issue.patch](https://storage.googleapis.com/google-code-attachments/skipfish/issue-9/comment-0/skipfish-color-issue.patch)

**Comment by GoogleCodeExporter** [comment]
```
The patch works fine. Thx!
```

Original comment by `Ozwo...@gmail.com` on 22 Mar 2010 at 7:30

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the patch, will have a look shortly.
```

Original comment by `lcam...@gmail.com` on 22 Mar 2010 at 3:37
- Added labels: **Priority-Low**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Now fixed in the upcoming 1.11.
```

Original comment by `lcam...@gmail.com` on 22 Mar 2010 at 3:53
- Changed state: **Fixed**
